src/main/java/org/fungover/mmotodo/rest/task/TaskController.java

```diff
@@ -0,0 +1,59 @@
+package org.fungover.mmotodo.rest.task;
+
+import org.fungover.mmotodo.entities.task.Task;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/tasks")
+public class TaskController {
+
+    private final TaskService service;
+
+    public TaskController(TaskService service) {
+        this.service = service;
+    }
+
+    @GetMapping("/")
+    public ResponseEntity<List<Task>> getAllTasks(@RequestParam(required = false) Long userId,
+                                                  @RequestParam(required = false) Long teamId) {
```

Comment:
Here you only need to have return type List<Task> without the need of ResponseEntity, as Spring Boot does auto convert to json & 200 OK status. Will reduce amount of code a bit :)
```java
public List<Task> getAllTasks() {}
```

Reply:
That is neat, going to trim the responses to be slimmer.

Reply:
Unnecessary ResponseEntities are now trimmed.